ext/web/08_text_encoding.js

```diff
@@ -91,40 +93,41 @@
           allowShared: true,
         });
       }
-      options = webidl.converters.TextDecodeOptions(options, {
-        prefix,
-        context: "Argument 2",
-      });
+      let stream = false;
+      if (options !== undefined) {
+        options = webidl.converters.TextDecodeOptions(options, {
+          prefix,
+          context: "Argument 2",
+        });
+        stream = options.stream;
+      }
```

Comment:
This change should be made into a more general change in the WebIDL dictionary converter. That way it automatically applies to all WebIDL dictionary converters, not just this one.